lib/pbench/test/unit/agent/task/test_make_result_tb.py

```diff
@@ -53,36 +51,34 @@ def config_and_logger(self):
                 self.result_dir,
                 self.target_dir,
                 self.config,
-                self.logger,
             ) = (
                 None,
                 None,
                 None,
                 None,
                 None,
-                None,
             )
```

Comment:
This construct is cute when it fits on one line; here it is ridiculous.  We should replace it with either
```
                self.controller = None
                self.name = None
                self.result_dir = None
                self.target_dir = None
                self.config = None
```
or, perhaps something like this
```
            (
                self.controller,
                self.name,
                self.result_dir,
                self.target_dir,
                self.config,
            ) = (None,) * 5
```
but that latter still looks ridiculous relative to the former.

Reply:
Sure.

Reply:
Fixed.